src/Batch/Batch/help/New-AzBatchResourceFile.md

```diff
@@ -0,0 +1,205 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Batch.dll-Help.xml
+Module Name: Az.Batch
+online version: https://docs.microsoft.com/en-us/powershell/module/az.batch/new-azbatchresourcefile
+schema: 2.0.0
+---
+
+# New-AzBatchResourceFile
+
+## SYNOPSIS
+Creates a Resource File for usage by `New-AzBatchTask`.
+
+## SYNTAX
+
+### HttpUrl (Default)
+```
+New-AzBatchResourceFile -HttpUrl <String> -FilePath <String> [-FileMode <String>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### StorageContainerUrl
+```
+New-AzBatchResourceFile [-FilePath <String>] [-FileMode <String>] [-BlobPrefix <String>]
+ -StorageContainerUrl <String> [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### AutoStorageContainerName
+```
+New-AzBatchResourceFile [-FilePath <String>] [-FileMode <String>] -AutoStorageContainerName <String>
+ [-BlobPrefix <String>] [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Creates a Resource File for usage by `New-AzBatchTask`.
+
+## EXAMPLES
+
+### Example 1: Create a resource file from an HTTP URL pointing at a single file
+```
+PS C:\> $file = New-AzBatchResourceFile -HttpUrl "https://testacct.blob.core.windows.net/" -FilePath "file1"
+New-AzBatchTask -JobId "Job-000001" -Id "Task23" -CommandLine "cmd /c dir /s" -ResourceFiles $file -BatchContext $Context
```

Comment:
@matthchr nit: for each of these examples, would you mind providing the resulting output that the user would see when running the command?

Reply:
Actually the specific commands shared won't show any output. Neither `New-AzBatchResourceFile` or `New-AzBatchTask` log anything by default. If you turned on the `-Debug` or `-Verbose` you'd see things but that's not how the sample is set up, so I think we're okay here, since the help file is showing what'd you see when you ran the commands (which is nothing)